### PR TITLE
Document a bunch of info tables (based on comments mainly)

### PR DESCRIPTION
**Pull request by Feacur**
general reasoning:
- it's easier to see actual constants than read comments
- comments were only in one place, but constants are ubiqutous
  - (some comments are preserved, though; did't want to inflate names)

concerning the dialogues with Gorons @ `z_en_go.c`:
- these inf table flags are used to not repeat certain messages
- so, the main purpose of new name is to convey the fact that they've said something already
- definitely, naming can be improved
  - say, Ingo offers Link a ride after they've met for the first time
    ```
    in that case `INFTABLE_9A` for message `0x2030` "do you want to ride one of my fine horses?"
    might be be something like `INFTABLE_TALKED_TO_INGO_FIRST_TIME`, like it was done for Malon
    ```
  - but sometimes it's a bit of a hassle
    ```
    the purpose of `INFTABLE_10F` for messages
    - `0x3041` "Dad and you destroyed the dragon together, didn't you!"
    - `0x3042` "When I grow up, I want to be a strong man like you"
    is to eliminate repeated opening lines. all's good, but `INFTABLE_TALKED_TO_GORON_FIRST_TIME`
    would be a bad name, because the actor profile is shared between entities. i need to find a
    unique name for each of the cases rather than resort to "the-one-over-there-before-an-event"
    these info is better placed as comments so that LSP would tell you any extended meaning

    meanwhile `INFTABLE_TALKED_TO_GORON_MSG_3041` or `INFTABLE_HEARD_MSG_3041` should do, i think

    `INFTABLE_EB` might be `INFTABLE_KNOW_ABOUT_BOMB_FLOWERS`, but it can be done later, given the
    change makes a difference
    ```

one way or another will conflict a bit with https://github.com/zeldaret/oot/pull/2299 at some info tables. negligible, just for me to keep in mind

### CODE REVIEW

File: include/z64save.h [hunk 1]
```diff
@@ -502,39 +502,39 @@ typedef enum LinkAge {
 #define EVENTCHKINF_33 0x33
 #define EVENTCHKINF_37 0x37
 #define EVENTCHKINF_38 0x38
-#define EVENTCHKINF_39 0x39
-#define EVENTCHKINF_3A 0x3A
+#define EVENTCHKINF_OPENED_ZORA_DOMAIN 0x39
```
Comment (cadmic) [inline include/z64save.h:505]: nit: `ZORAS_DOMAIN`

File: include/z64save.h [hunk 2]
```diff
@@ -797,24 +797,24 @@ typedef enum LinkAge {
...
+#define INFTABLE_TALKED_TO_GORON_MSG_3018 0xF8 // outside Darunias door (after opening door, before getting goron bracelet)
 #define INFTABLE_FC 0xFC
-#define INFTABLE_109 0x109
-#define INFTABLE_10A 0x10A
-#define INFTABLE_10B 0x10B
-#define INFTABLE_10C 0x10C
-#define INFTABLE_10D 0x10D
-#define INFTABLE_10E 0x10E
-#define INFTABLE_10F 0x10F
+#define INFTABLE_GOT_FIRE_TUNIC 0x109 // from Goron Link
```
Comment (cadmic) [inline include/z64save.h:808]: GORON_TUNIC

File: include/z64save.h [hunk 3]
```diff
@@ -770,7 +770,7 @@ typedef enum LinkAge {
 #define INFTABLE_AB 0xAB
 #define INFTABLE_B0 0xB0
 #define INFTABLE_B1 0xB1
-#define INFTABLE_B4 0xB4
+#define INFTABLE_TALKED_TO_GORON_MSG_3054 0xB4
```
Comment (cadmic) [inline include/z64save.h:773]: (I feel like these names should be more specific than `INFTABLE_TALKED_TO_GORON_MSG_3054`, but it's not worse than before so it's whatever)

**Review by cadmic — APPROVED** [review]

File: include/z64save.h (hunk 3 above)
Comment (Feacur) [reply include/z64save.h:773]: yeah, probably i should shuffle through the messages more closely

Comment (Feacur) [reply include/z64save.h:773]: ok, gotcha

File: include/z64save.h (hunk 2 above)
Comment (Feacur) [reply include/z64save.h]: changed with 78849051e9dfad5390192b37f0840007c2f3517f

File: include/z64save.h (hunk 1 above)
Comment (Feacur) [reply include/z64save.h]: changed with f369482f6df6c0ff7311c821dd630acac441de4f

**Comment by Feacur** [comment]
in the light of more atomic PRs for basically the same flags, better to close this one

**Pull request closed without merging**